src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age && <p className="Person__age">I am {person.age}</p>}
+    {person.isMarried ? (
```

Comment:
Add empty line between 
```suggestion
    {person.age && <p className="Person__age">I am {person.age}</p>}
    
    {person.isMarried ? (
```